src/common/_buttons.scss

```diff
@@ -173,6 +173,11 @@
       background-color: $close-button-active-background-color;
     }
   }
+
+  &[disabled] {
+    filter: grayscale(1.0) opacity(0.7);
```

Comment:
Not sure, but isn't it prefered to use opacity: 0.7 (top level rule) rather than a CSS filter?

Reply:
Also not sure, whether grayscale is neccessary.

Reply:
Eh, this seemed right to me. It gives the button the right look of being unavailable. (and filter vs rule... no functional difference)